pages/about.vue

```diff
@@ -44,31 +44,35 @@
       </v-row>
 
       <!-- We value row -->
-      <v-row class="mb-10 pa-12" id="we-value">
+      <v-row :class="pageSectionClass" id="we-value">
         <h2>We Value</h2>
 
         <v-row>
           <v-col
-            cols="10"
+            cols="12"
             :md="4"
-            :sm="10"
+            :sm="12"
```

Comment:
Don't think you need this- columns will automatically roll over from xs (set with `cols`) to sm